agent/consul/server.go

```diff
@@ -150,9 +150,9 @@ type Server struct {
 	// Enterprise user-defined areas.
 	router *servers.Router
 
-	// rpcListener is used to listen for incoming connections
-	rpcListener net.Listener
-	rpcServer   *rpc.Server
+	// Listener is used to listen for incoming connections
+	Listener  net.Listener
```

Comment:
Not really but the DNS server is doing that and I don't see harm in it. Don't have a strong opinion as long as the code that needs to get the real bind address can.